openfisca_web_api_preview/app.py

```diff
@@ -34,13 +34,15 @@ def init_tracker(url, idsite):
         log.warn(message)
 
 
-def create_app(country_package = os.environ.get('COUNTRY_PACKAGE'),
-               tracker_url = os.environ.get('TRACKER_URL'),
-               tracker_idsite = os.environ.get('TRACKER_IDSITE')):
+def create_app(country_package,
+               extensions = None,
+               tracker_url = None,
+               tracker_idsite = None):
+
     if country_package is None:
         raise ValueError(
-            u"You must specify a country package to start the API. "
-            u"For instance, `COUNTRY_PACKAGE=openfisca_france flask run`"
+            u"You must specify a country package with `-c` option to start the API. "
```

Comment:
Prefer long options (`--contry_template`) for readability

Reply:
But a regular user has only one country package installed on their environment. Why not attempt a [country package detection](https://github.com/openfisca/openfisca-core/blob/master/openfisca_core/scripts/__init__.py#L29) ?